stdlib/2and3/builtins.pyi

```diff
@@ -0,0 +1,1575 @@
+    def issuperset(self, s: Iterable[Any]) -> bool: ...
+    def pop(self) -> _T: ...
+    def remove(self, element: _T) -> None: ...
+    def symmetric_difference(self, s: Iterable[_T]) -> Set[_T]: ...
+    def symmetric_difference_update(self, s: Iterable[_T]) -> None: ...
+    def union(self, *s: Iterable[_T]) -> Set[_T]: ...
+    def update(self, *s: Iterable[_T]) -> None: ...
+    def __len__(self) -> int: ...
+    def __contains__(self, o: object) -> bool: ...
+    def __iter__(self) -> Iterator[_T]: ...
+    def __str__(self) -> str: ...
+    def __and__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __iand__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __or__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
+    def __ior__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
+    def __sub__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __isub__(self, s: AbstractSet[object]) -> Set[_T]: ...
+    def __xor__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
+    def __ixor__(self, s: AbstractSet[_S]) -> Set[Union[_T, _S]]: ...
+    def __le__(self, s: AbstractSet[object]) -> bool: ...
+    def __lt__(self, s: AbstractSet[object]) -> bool: ...
+    def __ge__(self, s: AbstractSet[object]) -> bool: ...
+    def __gt__(self, s: AbstractSet[object]) -> bool: ...
+
+class frozenset(AbstractSet[_T], Generic[_T]):
+    def __init__(self, iterable: Iterable[_T] = ...) -> None: ...
+    def copy(self) -> FrozenSet[_T]: ...
+    def difference(self, *s: Iterable[object]) -> FrozenSet[_T]: ...
+    def intersection(self, *s: Iterable[object]) -> FrozenSet[_T]: ...
+    def isdisjoint(self, s: Iterable[_T]) -> bool: ...
+    def issubset(self, s: Iterable[object]) -> bool: ...
+    def issuperset(self, s: Iterable[object]) -> bool: ...
+    def symmetric_difference(self, s: Iterable[_T]) -> FrozenSet[_T]: ...
+    def union(self, *s: Iterable[_T]) -> FrozenSet[_T]: ...
+    def __len__(self) -> int: ...
+    def __contains__(self, o: object) -> bool: ...
+    def __iter__(self) -> Iterator[_T]: ...
+    def __str__(self) -> str: ...
+    def __and__(self, s: AbstractSet[_T]) -> FrozenSet[_T]: ...
+    def __or__(self, s: AbstractSet[_S]) -> FrozenSet[Union[_T, _S]]: ...
+    def __sub__(self, s: AbstractSet[_T]) -> FrozenSet[_T]: ...
+    def __xor__(self, s: AbstractSet[_S]) -> FrozenSet[Union[_T, _S]]: ...
+    def __le__(self, s: AbstractSet[object]) -> bool: ...
+    def __lt__(self, s: AbstractSet[object]) -> bool: ...
+    def __ge__(self, s: AbstractSet[object]) -> bool: ...
+    def __gt__(self, s: AbstractSet[object]) -> bool: ...
+
+class enumerate(Iterator[Tuple[int, _T]], Generic[_T]):
+    def __init__(self, iterable: Iterable[_T], start: int = ...) -> None: ...
+    def __iter__(self) -> Iterator[Tuple[int, _T]]: ...
+    if sys.version_info >= (3,):
+        def __next__(self) -> Tuple[int, _T]: ...
+    else:
+        def next(self) -> Tuple[int, _T]: ...
+    # TODO __getattribute__
```

Comment:
This can be removed. enumerate objects do have a `__getattribute__`, but from looking at the C code it doesn't seem like this creates any real attributes.